core/whistle_number_manager-1.0.0/src/wh_number_fix.erl

```diff
@@ -0,0 +1,334 @@
+%%%-------------------------------------------------------------------
+%%% @copyright (C) 2015, 2600Hz INC
+%%% @doc
+%%%
+%%% @end
+%%% @contributors
+%%%   Peter Defebvre
+%%%-------------------------------------------------------------------
+-module(wh_number_fix).
+
+-export([fix_account_numbers/1]).
+
+-include("wnm.hrl").
+
+-define(PHONE_NUMBERS, <<"phone_numbers">>).
+
+-define(TIME_BETWEEN_ACCOUNTS, whapps_config:get_integer(?WNM_CONFIG_CAT, <<"time_between_accounts">>, 1000)).
```

Comment:
Add a suffix to the keys for the unit of time:
*  "_s" -> seconds
* "_ms" -> milliseconds